tools/inkscape_extension/visicut_export.py

```diff
@@ -336,21 +337,38 @@ def get_original_filename(filename):
 
 # Try to start own VisiCut instance
 try:
-    creationflags = 0
-    close_fds = False
     if os.name == "nt":
         DETACHED_PROCESS = 8  # start as "daemon"
         creationflags = DETACHED_PROCESS
         close_fds = True
+
+        cmd = [VISICUTBIN] + arguments + [dest_filename]
+        subprocess.Popen(cmd, creationflags=DETACHED_PROCESS, close_fds=True)
+
     else:
-        try:
-            import daemonize
-            daemonize.createDaemon()
-        except:
-            sys.stderr.write("Could not daemonize. Sorry, but Inkscape was blocked until VisiCut is closed")
-    cmd = [VISICUTBIN] + arguments + [dest_filename]
-    Popen(cmd, creationflags=creationflags, close_fds=close_fds)
-except:
-    sys.stderr.write("Can not start VisiCut (" + str(sys.exc_info()[0]) + "). Please start manually or change the VISICUTDIR variable in the Inkscape-Extension script\n")
+        # on POSIX-compliant OSes (Linux, macOS etc.) we can detach from the parent process using the classic double
```

Comment:
Your change removes the need for `daemonize.py`. Therefore that file should be dropped.
https://github.com/t-oster/VisiCut/blob/master/tools/inkscape_extension/daemonize.py

Reply:
I see. I thought this was a third-party dependency to `pip install`.